src/test/java/TestsWithRetrofit/TrelloRestApiTest.java

```diff
@@ -0,0 +1,79 @@
+package TestsWithRetrofit;
+
+import TestsWithRestAssured.BoardForTests;
+import okhttp3.OkHttpClient;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.testng.Assert.assertEquals;
+
+public class TrelloRestApiTest {
+
+    String key;
+    String token;
+    Retrofit retrofit;
+    List<String> idListOfCreatedBoards;
+
+    @BeforeClass
+    public void init(){
+
+        idListOfCreatedBoards = new ArrayList<>();
+        Properties properties = new Properties();
+        String pathToFile = "src/main/resources/trello.properties";
+
+        try {
+            properties.load(new FileInputStream(pathToFile));
+            key = properties.getProperty("key");
+            token = properties.getProperty("token");
+        }
+        catch (IOException ex){
+            ex.printStackTrace();
+        }
+
+        OkHttpClient.Builder httpClient = new OkHttpClient.Builder();
+        retrofit = new Retrofit.Builder()
+                .baseUrl("https://trello.com")
+                .addConverterFactory(GsonConverterFactory.create())
+                .client(httpClient.build())
+                .build();
+    }
+
+    @Test
+    public void creatingNewBoardTest(){
+
+        BoardForTests board = new BoardForTests();
+        board.setName("NewBoard");
+        board.setDesc("This is new board!");
+        board.setClosed(false);
+
+        BoardService service = retrofit.create(BoardService.class);
+        Call<BoardForTests> callSync = service.putBoard(key, token, board);
+
+        try {
```

Comment:
you don't need to handle this exception with try/catch. please add exception to method signature